mws/acceptance/mws_permissionassignments_test.go

```diff
@@ -0,0 +1,64 @@
+package acceptance
+
+import (
+	"testing"
+
+	"github.com/databricks/terraform-provider-databricks/internal/acceptance"
+	"github.com/databricks/terraform-provider-databricks/qa"
+)
+
+func TestAccAssignGroupToWorkspace(t *testing.T) {
```

Comment:
Just curious, what does this test do? Does it test that the template can be accepted?

Reply:
it does create, read, update, and delete test assignments of test principals